Card/card.css

```diff
@@ -0,0 +1,59 @@
+.card-e-commerce {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1.5rem;
+    margin-bottom: 6rem;
+}
+
+.hidden {
+    display: none;
+}
+
+.card-e-commerce .item-image {
+    width: 10rem;
+    height: 15rem;
+}
+
+.item-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    gap: 0.5rem;
+    width: max-content;
+}
+
+.item-author {
+    font-size: 0.8rem;
+    color: var(--secondary-color)
+}
+
+.item-price {
+    font-size: 1.2rem;
+}
+
+.button-group {
+    display: flex;
+    flex-direction: column;
+    gap: 0.75rem;
+    margin-top: 1rem;
+}
+
+.button-group button {
+    height: 40px;
```

Comment:
Check if you could use rem here

Reply:
Done. Thanks, all the suggestions incorporated.